content/_code-samples/claim-payment-channel/README.md

```diff
@@ -0,0 +1,5 @@
+# Create and Claim a Payment Channel
+
+Create, claim and verify a Payment Channel.
+
+For more context, see [PayChannel](https://xrpl.org/paychannel.html#paychannel).
```

Comment:
Make sure to add these summaries in comments to the top of the corresponding code snippet :)